hax/hax/util.py

```diff
@@ -1710,20 +1710,26 @@ def get_service_health(self,
     @repeat_if_fails()
     @uses_consul_cache
     def get_process_node(self, proc_fid: Fid, kv_cache=None) -> str:
-        fidk = proc_fid.key
-        # 'node/<node_name>/process/<process_fidk>/service/type'
-        # node_items = self.kv.kv_get('m0conf/nodes',
-        #                             recurse=True,
-        #                             kv_cache=kv_cache)
-        node_items = self.get_all_nodes(kv_cache=kv_cache)
-        if ObjT.PROCESS.value == proc_fid.container:
-            keys = self.get_process_keys(node_items, fidk)
-        elif ObjT.SERVICE.value == proc_fid.container:
-            keys = self.get_service_keys(node_items, fidk)
-        key = keys[0].split('/')
-        node_key = ('/'.join(key[:3]))
-        node_val = self.kv.kv_get(node_key, kv_cache=kv_cache)
-        data = node_val['Value']
+        try:
+            fidk = proc_fid.key
+            # 'node/<node_name>/process/<process_fidk>/service/type'
+            # node_items = self.kv.kv_get('m0conf/nodes',
+            #                             recurse=True,
+            #                             kv_cache=kv_cache)
+            node_items = self.get_all_nodes(kv_cache=kv_cache)
+            if ObjT.PROCESS.value == proc_fid.container:
+                keys = self.get_process_keys(node_items, fidk)
+            elif ObjT.SERVICE.value == proc_fid.container:
+                keys = self.get_service_keys(node_items, fidk)
+            LOG.debug('proc_fid: %s keys: %s', proc_fid, keys)
+            if not keys:
+                raise HAConsistencyException('Failed to get process node')
+            key = keys[0].split('/')
+            node_key = ('/'.join(key[:3]))
+            node_val = self.kv.kv_get(node_key, kv_cache=kv_cache)
```

Comment:
In kv_get function we have `LOG.debug('KVGET key=%s, kwargs=%s', key, kwargs)`
Should we also add similar log for 'Result' in same function after read is compete.